client/scss/components/_body_content.scss

```diff
@@ -45,3 +45,54 @@
     }
   }
 }
+
+$large-width: (5/7 * 100)+'%';
```

Comment:
What's the difference between `$large-width` and `$narrow-width`?

Could/should we rewrite these using [Sass percentages](http://sass-lang.com/documentation/Sass/Script/Functions.html#percentage-instance_method)?